synapse/lib/stormtypes.py

```diff
@@ -212,7 +212,7 @@ async def _libPkgAdd(self, pkgdef):
         Returns:
             dict: The validated storm package definition.
         '''
-        self.runt.user.confirm(('pkgs', 'add'), None)
+        self.runt.user.confirm(('pkg', 'add'), None)
```

Comment:
????

Reply:
storm package add/remove permissions were 'pkg' in cortex.py and 'pkgs' here, which I'm assuming was not intentional.